### ISSUE TITLE
Suggestion: can goreman be run as a "daemon" 

### ISSUE THREAD

**Issue by liuhewei** [mattn/goreman#17]
Sometimes I want to control these processes to be started one by one, rather than start parallel. But I can't handle it easily now:

I can use `goreman run start [process]`, but goreman process itself must be started before. Since goreman hasn't a "daemon" model, it seems that I have to run `goreman start [process] &`, and then use `goreman run ...`.

I wonder if goreman have a "daemon" model for running itself. For example:
```
goreman start -d  # start nothing but itself, as a daemon process
goreman run start ....  # use "goreman run" command
goreman stop   # stop goreman daemon process itself
```

What do you think abou this?

**Comment by csrutil** [comment]
@liuhewei Yeah, I want this feature too, there is a PM2 in nodejs world has the daemon mode, but it's too heavy
